docs/TELEMETRY.md

```diff
@@ -23,35 +23,75 @@
-
-* UI
-
-* Interactions
+Pings should be sent from the addon in the following cases:
+* Study enrollment / initialization (`start` pingType)
+* Study unenrollment / expiration (`stop` pingType)
+* UI interaction with the addon (`engage` pingType)
 
-All interactions with the UI create sequences of Telemetry Pings.
+Type of interaction events we need recorded (`interactionType` in ping schema):
 
-All UI `shield-study` `study_state` sequences look like this:
+1. `toolbarClick`: user clicked on the study's toolbar icon
+2. `signinClick`: user clicked on the sync signin button from the doorhanger (only shown when user is not signed in)
+3. `unverifiedOpenSyncClick`: user clicked on the button to open sync preferences (only shown when account is in unverified state)
+4. `verifiedOpenAccountClick`: user clicked on the button to open FxA account prefs (only shown when account is in verified state)
+5. `verifiedOpenSyncClick`: user clicked on the button to open sync prefs (only shown when account is in verified state)
+6. `verifiedAvatarClick`: user clicked on their avatar to change it (only shown when account is in verified state)
+7. `verifiedSendTabClick`: user clicked on the send tab link (TBD if this will be included; only shown in verified state; only shown to treatment2 branch)
 
-* `enter => install => exit`.
-
-## Example sequence
-
-These are the `payload` fields from all pings in the `shield-study` and `shield-study-addon` buckets.
+## Example ping schema could look like this
+This is just the schema for the specific things we need in this study (what goes in the payload section of the `shield-study-addon` ping), for an example of the full schema see [this](https://github.com/motin/taar-experiment-v3-shield-study/blob/develop/schemas/full-schema.json).
 
 ```
-// common fields
-
-branch        up-to-expectations-1        // should describe Question text
-study_name    57-perception-shield-study
-addon_version 1.0.0
-version       3
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "fxadisco",
+  "definitions": {
+    "attributes": {
+      "properties": {
+        "pingType": {
+          "type": "string" // one of: start, stop, engage
+        },
+        "addonId": {
+          "type": "string" // always fxadisco
+        },
+        "addonVersion" {
+          "type": "string" // should be 1
+        },
+        "branch" {
+          "type": "string" //  enrollment cohort; one of control, treatment1, treatment2
+        },
+        "startTime": {
+          "type": "string" // timestamp of interaction event or enrollment
+        },
+        "fxaState": {
+          "type": "string" // one of: verfied, unverfied, none
+        },
+        "interactionType": {
+          "type": "string" // see list above for possible values, or `none`
+        },
+        "doorhangerActiveSeconds": {
+          "type": "string" // number of seconds doorhanger was displayed for (0 if none)
+        },
+        "hasAvatar": {
```

Comment:
> I wonder if we could get away with sending the FxA user ID

We do have access to the UID if we decided to send it. Below are some of the other values that we have access to

```javascript
{
    email: "vbudhram+test@mozilla.com",
    locale: "en-US,en;q=0.5",
    amrValues: [],
    twoFactorAuthentication: false,
    uid: "5298b1da5fb64496a8a94a8c141ff99f",
    avatar: "https://firefoxusercontent.com/c3f000621a1717c9c29eb606590f8e41",
    avatarDefault: false,
    displayName: "sdfdd",
    verified: true
}
```

Not sure if `displayName` and `twoFactorAuthentication` would have any value here.